packages/button/README.md

```diff
@@ -80,19 +80,19 @@ npm install @leafygreen-ui/button
+| `variant`      | `'default'`, `'primary'`, `'info'`, `'danger'`, `'dark'` | Sets the style variant of the button.                                                                                                                                                                           | `'default'`                                                                                                                       |
+| `darkMode`     | `boolean`                                                | Determines if the component renders in dark mode                                                                                                                                                                | `false`                                                                                                                           |
+| `size`         | `'xsmall'`, `'small'`, `'default'`, `'large'`            | Sets the size variant of the button.                                                                                                                                                                            | `'default'`                                                                                                                       |
+| `children`     | `node`                                                   | The content that will appear inside of the `<Button />` component.                                                                                                                                              | `null`                                                                                                                            |
+| `className`    | `string`                                                 | Adds a className to the class attribute.                                                                                                                                                                        | `''`                                                                                                                              |
+| `disabled`     | `boolean`                                                | Disabled the button                                                                                                                                                                                             | `false`                                                                                                                           |
+| `as`           | `'a'                                                     | 'button'`                                                                                                                                                                                                       | Determines the root element. An `a` tags can be supplied to replace `button` from being the DOM element that wraps the component. | `button` |
```

Comment:
Heads up, using pipes in md table text creates a new column. We use commas to represent unions here (or use`\|`)

Reply:
i.e. `'a'  | 'button'` created an extra column